src/components/MenuButton/index.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react"
+import Dropdown from "../Dropdown"
+
+function MenuButton (){
+  const [isMenuOpen, setIsMenuOpen] = useState(false)
+
+  function toggleMenu() {
+    setIsMenuOpen((prevIsMenuOpen) => !prevIsMenuOpen)
```

Comment:
Doing `(prevIsMenuOpen) => !prevIsMenuOpen` is not needed - you can just do setIsMenuOpen(!isMenuOpen)